src/System.Drawing.Common/src/System/Drawing/BufferedGraphics.Windows.cs

```diff
@@ -6,18 +6,10 @@
 
 namespace System.Drawing
 {
-    /// <summary>
-    /// The BufferedGraphics class can be thought of as a "Token" or "Reference" to the buffer that a
-    /// BufferedGraphicsContext creates. While a BufferedGraphics is outstanding, the memory associated with the
-    /// buffer is locked. The general design is such that under normal conditions a single BufferedGraphics will be in
-    /// use at one time for a given BufferedGraphicsContext.
-    /// </summary>
-    public sealed class BufferedGraphics : IDisposable
+    public sealed partial class BufferedGraphics
```

Comment:
You removed the `: IDisposable` from the .Windows file but not from the .Unix file.  Why the inconsistency?